recipe/meta.yaml

```
@@ -38,11 +38,10 @@ requirements:
     - jedi >=0.16
     - matplotlib-inline
     - pexpect >4.3    # [unix]
```

This suggestion re-adds the pickleshare dependency.
```suggestion
    - pexpect >4.3    # [unix]
    - pickleshare
```